src/shared/read_parameter_file.F90

```diff
@@ -273,6 +273,18 @@ subroutine read_parameter_file()
   call read_value_logical(ADIOS_FOR_UNDO_ATTENUATION, 'ADIOS_FOR_UNDO_ATTENUATION', ier)
   if (ier /= 0) stop 'an error occurred while reading the parameter file: ADIOS_FOR_UNDO_ATTENUATION'
 
+#if !defined(HAVE_ADIOS2) && !defined(ADIOS_INPUT)
+    print *
```

Comment:
hi Norbert, 

could you add an if-statement here? at the moment, this always fails when configured without ADIOS option. this following code part should only execute with:
if (ADIOS_ENABLED) then
..
many thanks,
daniel